### PR TITLE
Update MODULES.yml

### PR DESCRIPTION
**Pull request by hereariim**
module for 2D segmentation of flower apple tree

### CODE REVIEW

File: MODULES.yml [hunk 1]
```diff
@@ -19,3 +19,4 @@
 - module: https://github.com/deephdc/DEEP-OC-neural_transfer
 - module: https://github.com/deephdc/DEEP-OC-image-classification-tf-dicom
 - module: https://github.com/deephdc/DEEP-OC-demo_app
+- module: https://github.com/deephdc/DEEP-OC-flower
```
Comment (IgnacioHeredia) [inline MODULES.yml:22]: This should be 
```
- module: https://github.com/hereariim/DEEP-OC-blossoming
```

**Comment by IgnacioHeredia** [comment]
@hereariim the url you should be the one of your DEEP-OC repo (see comment on file change).
You should also change the title and summary in the `metadata.json` to make it more descriptive. For example:
```json
    "title": "Apple tree blossoming",
    "summary": "Segmenting the flowers of an apple tree with an UNET",
```

**Comment by hereariim** [comment]
hello @IgnacioHeredia 
I added some correction. I wrote : 
`- module: https://github.com/hereariim/DEEP-OC-flower`
And, i wrote some corrections in `metadata.json` file (file located in DEEP-OC-flower) : 
`"title" : "Apple tree blossoming",`
`"summary":"Segmenting the flowers of an apple tree based on UNET algorithm"`

**Comment by IgnacioHeredia** [comment]
> I added some correction. I wrote :
> - module: https://github.com/hereariim/DEEP-OC-flower

You need to commit the changes in this branch (`patch-2`) so that they are reflected in the Pull Request.

**Pull request closed without merging**